src/os/macos.rs

```diff
@@ -16,6 +16,9 @@ pub trait WindowExt {
     ///
     /// The pointer will become invalid when the `Window` is destroyed.
     fn get_nsview(&self) -> *mut c_void;
+
+    /// Request user attention. Only relevant on macOS.
```

Comment:
I put the `Only relevant on macOS` bit in because that is what the NIX-specific code does for X11 (though now I realize you can have non-X11NIX, e.g. wayland) : https://github.com/tomaka/winit/blob/bc03ffb3179d0c932b981d8cc2f479e900492073/src/os/unix.rs#L106 but I can remove that.

Reply:
Yeah, removing that would be good, since AFAIK it will never be an issue.